init.c

```diff
@@ -387,6 +409,8 @@ int main(int argc, char *argv[])
 		goto loop;
 	}
 
+	load_early_drivers(pv_config_get_system_mdev());
```

Comment:
I this as late in the init process as it can be? I guess it cannot be added to pv_init_tbl? In that case, do we want this to be executed in IM_STANDALONE and IM_APPENGINE? If so, maybe we should control what we are loading to then be able to unload it during shutdown. Either way, remember to test this in appengine when you get it ready :)

Reply:
> I this as late in the init process as it can be? I guess it cannot be added to pv_init_tbl?

I guess this should be before to call `other_mounts()` because we need the fs driver and probably the `usb-storage`. 

> do we want this to be executed in IM_STANDALONE and IM_APPENGINE?

could be... but that'd be another target? something like `x64-ubuntu-lts-appengine` ??

Reply:
I think there is no need to add is as a target for now. I was more interested in what new stuff is going to be executed here for the existing appengine targets  (and for other targets now that I think about it). I guess, according to the "system.mdev" parameter, we are going to still do "pv_drivers_load_single("usb-storage", NULL);" for the targets that have it disabled. Is this what we want?